README.md

```diff
@@ -345,6 +345,17 @@ protected override void OnAfterRender(bool firstRender)
 </DateTimePicker>
 ```
 
+To achieve the same behavior as in the original library (the current date is preselected) the following code can be used:
+
+```html
+<DateTimePicker 
+        DateChangeEvent="(date) => DateChangeEventTest(date)" 
+        From=@DateTime.Now.ToString("yyyy/MM/dd", CultureInfo.InvariantCulture)
```

Comment:
It's already defined in [line 20](https://github.com/yagizhanNY/siemens-ix-blazor/blob/896446eced09869a31760560b6db3153a0e26a92/SiemensIXBlazor/Components/DateTimePicker/DateTimePicker.razor.cs#L20). Is it now working as expected?

Reply:
`CultureInfo.InvariantCulture` is missing I think, coz I using the german culture settings

Reply:
Hi @DaveHOnCode, will you change the code behind?

Reply:
Yeah I can see about it if I'll get more time